### PR TITLE
Porting Inland mk47 from Via

### PR DESCRIPTION
**Pull request by usrfriendly**
This is a quick, effective port from Via to Vial for the Inland MK47 keyboard.  It appears to be fully featured and fully functional with only enough modification to add the required files/configurations Vial needs, simply following the porting guide, compiles small enough to fit on the EEPROM.

### CODE REVIEW

File: keyboards/inland/mk47/keymaps/vial/vial.json [hunk 1]
```diff
@@ -0,0 +1,156 @@
+{
+    "name": "MK47",
+    "vendorId": "0xFFFE",
+    "productId": "0x0002",
+    "keycodes": ["qmk_lighting"],
```
Comment (lesshonor) [inline keyboards/inland/mk47/keymaps/vial/vial.json]:
```suggestion
```
VIA V3 setting that does nothing in Vial.

File: keyboards/inland/mk47/keymaps/vial/vial.json [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                "showIf": "{id_qmk_rgb_matrix_effect} != 0",
+                "label": "Color",
+                "type": "color",
+                "content": ["id_qmk_rgb_matrix_color", 3, 4]
+              }
+            ]
+          }
+        ]
+      }
+    ],
```
Comment (lesshonor) [inline keyboards/inland/mk47/keymaps/vial/vial.json]:
```suggestion
```
VIA V3 settings that do nothing in Vial.

**Review by lesshonor — CHANGES_REQUESTED** [review]

File: keyboards/inland/mk47/keymaps/vial/vial.json (hunk 2 above)
Comment (usrfriendly) [reply keyboards/inland/mk47/keymaps/vial/vial.json]: Sorry, it was a little _too_ quick and dirty.  I made the edits, squashed the commits and pushed to keep things clean, and did a test-build/flash on my board where everything worked fine.

thanks @lesshonor

File: keyboards/inland/mk47/keymaps/vial/vial.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
+{
+    "name": "MK47",
+    "vendorId": "0xFFFE",
+    "productId": "0x0002",
```
Comment (lesshonor) [inline keyboards/inland/mk47/keymaps/vial/vial.json]:
```suggestion
```
These technically aren't needed for Vial either, but doesn't really matter.

**Review by lesshonor — APPROVED** [review]

File: keyboards/inland/mk47/keymaps/vial/config.h [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# pragma once
```
Comment (adophoxia) [inline keyboards/inland/mk47/keymaps/vial/config.h:3]:
```suggestion
/* SPDX-License-Identifier: GPL-2.0-or-later */

# pragma once
```
Missing copyright header.

**Review by adophoxia — CHANGES_REQUESTED** [review]

**Comment by usrfriendly** [comment]
Tweaked to add licensing and clean up the unnecessary identifiers, did a test build on my end, and it's testing after push.

**Review by lesshonor — APPROVED** [review]

**Review by adophoxia — APPROVED** [review]

**Pull request merged**